### PR TITLE
Fix nft billboard

### LINKED ISSUE #874: [Examples] - Polish and needed checks
# Apps - Small Review

[protocol-monorepo/examples/employment-based-loan/contracts at dev · superfluid-finance/protocol-monorepo · GitHub](https://github.com/superfluid-finance/protocol-monorepo/tree/dev/examples/employment-based-loan/contracts)

- No checking if caller is host or with the right agreement.
- If not needed don’t save CFA and host if using CFALib (already point this in older PR: https://github.com/superfluid-finance/protocol-monorepo/pull/772)

[protocol-monorepo/BudgetNFT.sol at dev · superfluid-finance/protocol-monorepo · GitHub](https://github.com/superfluid-finance/protocol-monorepo/blob/dev/examples/budget-nft/contracts/BudgetNFT.sol)

- Not using CFALib
- Why Migrations.sol file?

[protocol-monorepo/examples/nftbillboard-userdata at dev · superfluid-finance/protocol-monorepo · GitHub](https://github.com/superfluid-finance/protocol-monorepo/tree/dev/examples/nftbillboard-userdata)
- Remove .github folder
- If not needed don’t save CFA and host if using CFALib (already talked about this in older PR: https://github.com/superfluid-finance/protocol-monorepo/pull/772)

[protocol-monorepo/DividendRightsToken.sol at dev · superfluid-finance/protocol-monorepo · GitHub](https://github.com/superfluid-finance/protocol-monorepo/blob/dev/examples/rewards-distribution-token/contracts/DividendRightsToken.sol)

- No checking if caller is host or with the right agreement.
- If not needed don’t save HOST if using IDALib

### PR DESCRIPTION
**Pull request by saflamini**
cc @ngmachado 

1) Unncessary state vars removed
2) .github folder removed.

Note: this example needs other TLC. In a future PR, I will
1) Remove bloat provided by scaffold eth
2) Re write test suite to SDK core (I missed re writing this one in our bulk re writes)
3) Edit scripts such that they're using our sdks instead of older code

### CODE REVIEW

File: examples/nftbillboard-userdata/packages/hardhat/contracts/RedirectAll.sol [hunk 1]
```diff
@@ -28,8 +28,6 @@ import {
 
 contract RedirectAll is SuperAppBase {
```
Comment (ngmachado) [inline examples/nftbillboard-userdata/packages/hardhat/contracts/RedirectAll.sol:24]: CFA Hash identifier: Use the same pattern when possible.
Remove unused imports

**Review by ngmachado — CHANGES_REQUESTED** [review]

File: examples/nftbillboard-userdata/packages/hardhat/contracts/RedirectAll.sol (hunk 1 above)
Comment (saflamini) [reply examples/nftbillboard-userdata/packages/hardhat/contracts/RedirectAll.sol:24]: @ngmachado this has been fixed and is ready for your review

**Review by ngmachado — APPROVED** [review]
LGTM

**Comment by saflamini** [comment]
> LGTM

Hey @ngmachado - I made several changes per our convo today.

1) Re built test suite using sdk core
2) Modernized deploy & interaction scripts
3) Removed many unneccessary packages and files - particularly within the react app 

@hellwolf - it looks like we may need to manually trigger the build-examples workflow for this to be merged with full confidence though. I have run the build-and-test.sh script locally with no issues, but want to make sure that it passes here too. Do you think you could manually trigger it?

**Comment by hellwolf** [comment]
strange, let me check why the CODEOWNERS file didn't work for fte team

**Comment by hellwolf** [comment]
aha, because there is a .github directory deleted from the example projects, and CODEOWNERS file should be amended to reflect only root level folders

**Review by hellwolf — APPROVED** [review]

**Pull request merged**